splinter/driver/webdriver/firefox.py

```diff
@@ -40,7 +40,7 @@ def __init__(
         firefox_capabilities = DesiredCapabilities().FIREFOX
         firefox_capabilities["marionette"] = True
 
-        firefox_options = Options()
+        options = Options() if options is None else options
```

Comment:
It is necessary to add the option as parameter in the `__init__` method definition.

And we can simplify this code using `or` boolean operator:
```suggestion
        options = options or Options()
```

Reply:
Add this simplification to both the firefox and chrome drivers.